native/src/main/java/io/ballerina/stdlib/udp/nativeclient/Client.java

```diff
@@ -44,57 +46,57 @@ public final class Client {
     private Client() {}
 
     public static Object init(Environment env, BObject client, BMap<BString, Object> config) {
-        final Future balFuture = env.markAsync();
-
-        BString host = config.getStringValue(StringUtils.fromString(Constants.CONFIG_LOCALHOST));
-        InetSocketAddress localAddress;
-        if (host == null) {
-            // A port number of zero will let the system pick up an ephemeral port in a bind operation.
-            localAddress = new InetSocketAddress(0);
-        } else {
-            localAddress = new InetSocketAddress(host.getValue(), 0);
-        }
-
-        double timeout = ((BDecimal) config.get(StringUtils.fromString(Constants.CONFIG_READ_TIMEOUT))).floatValue();
-        client.addNativeData(Constants.CONFIG_READ_TIMEOUT, timeout);
-
-        UdpClient udpClient = UdpFactory.getInstance().createUdpClient(localAddress, balFuture);
-        client.addNativeData(Constants.CONNECTIONLESS_CLIENT, udpClient);
-
-        return null;
+        return env.yieldAndRun(() -> {
```

Comment:
replace markAsync with env.yieldAndRun and completableFuture for Ballerina extern fucntions